.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+- '2.7'
+install: pip install -r requirements.txt --use-mirrors
+script: nosetests
```

Comment:
It seems like people try to wrap test execution (regardless of what tool is used) in `setup.py`. That way you don't really have to know any project specific stuff and just run `python setup.py test` to get the test suite going.

I opened https://github.com/WikiWatershed/tr-55/issues/23 to wire that up.